mkdocs.yml

```diff
@@ -1,7 +1,7 @@
 # Project information
-site_name: Playing the Internship Game
-site_description: 'A tutorial to python'
-site_author: 'Evy Kassirer'
+site_name: A Guide To All Things Internships
```

Comment:
site_name: Playing the Internship Game

site_description: A guide to all things internships